ui-test/src/main/java/regclient/pages/english/ApplicantBiometricsPageEnglish.java

```diff
@@ -191,72 +186,86 @@ public BiometricDetailsPage clickOnBackButton() {
 	}
 
 	@SuppressWarnings("deprecation")
-	public  boolean isApplicantBiometricsPageDisplyed() {
-		return isElementDisplayed(findElementWithRetry(MobileBy.AndroidUIAutomator("new UiScrollable(new UiSelector().scrollable(true).instance(0)).scrollIntoView(new UiSelector().descriptionContains(\"" + FetchUiSpec.getValueUsingId("individualBiometrics") + "\"))")));
+	public boolean isApplicantBiometricsPageDisplyed() {
+		return isElementDisplayed(findElementWithRetry(MobileBy.AndroidUIAutomator(
+				"new UiScrollable(new UiSelector().scrollable(true).instance(0)).scrollIntoView(new UiSelector().descriptionContains(\""
+						+ FetchUiSpec.getValueUsingId("individualBiometrics") + "\"))")));
 	}
 
 	@SuppressWarnings("deprecation")
-	public  boolean isAuthenticationBiometricsPageDisplyed() {
-		return isElementDisplayed(findElementWithRetry(MobileBy.AndroidUIAutomator("new UiScrollable(new UiSelector().scrollable(true).instance(0)).scrollIntoView(new UiSelector().descriptionContains(\"" + FetchUiSpec.getValueUsingId("individualAuthBiometrics") + "\"))")));
+	public boolean isAuthenticationBiometricsPageDisplyed() {
+		return isElementDisplayed(findElementWithRetry(MobileBy.AndroidUIAutomator(
+				"new UiScrollable(new UiSelector().scrollable(true).instance(0)).scrollIntoView(new UiSelector().descriptionContains(\""
+						+ FetchUiSpec.getValueUsingId("individualAuthBiometrics") + "\"))")));
```

Comment:
This is field Id hardcoding, this biometric type field id can be changed in the UI Spec